Sources/SpeziLLMOpenAI/FunctionCalling/LLMFunctionParameterSchemaCollector.swift

```diff
@@ -38,10 +36,16 @@ extension LLMFunction {
         
         let properties = schemaValueCollectors.compactMapValues { $0.schema }
         
-        return .init(
-            type: .object,
-            properties: properties,
-            required: requiredPropertyNames
-        )
+        var functionParameterSchema: LLMFunctionParameterSchema = .init()
+        do {
+            functionParameterSchema.additionalProperties = try .init(unvalidatedValue: [
+                "type": "object",
+                "properties": properties.mapValues { $0.value },
+                "required": requiredPropertyNames
+            ])
+        } catch {
+            logger.error("Error creating OpenAPIObjectContainer.")
```

Comment:
Is a fallthrough error handeling really beneficial here?
To be fair, errors should be very rare here, only when the validated container cannot be created

Reply:
`fatalError()`, then? Or can you think of a nicer way to handle this?